pom.xml

```
@@ -5,8 +5,9 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>mate-academy</groupId>
-    <artifactId>spring-rest</artifactId>
+    <artifactId>spring-web</artifactId>
```

```suggestion
    <artifactId>spring-rest</artifactId>
```
Let's revert the redundant configuration of your pom.xml